### DISCUSSION TITLE
Indexing Block Hanging ever 5k blocks

### DISCUSSION THREAD

**Discussion by walkermoz** [ordinals/ord#1619] [Q&A]
![image](https://user-images.githubusercontent.com/71871303/217710160-68d18472-fdb8-41ff-8ede-be05d7a40fb1.png)

When Indexing Blocks the cli hangs for 10 seconds - 3min every time. really increases the ordinals install time

**Comment by goddesslens** [comment]
That happens. But it is alright.  It will reconnect. No problem to inscribe
